.gitignore

```diff
@@ -8,4 +8,4 @@ stats.json
 npm-debug.log
 
 lib
-middleware
+/middleware
```

Comment:
should be `middleware/`

Reply:
I tried that, but it did not work as desired.

Reply:
See https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns for more info on why

Reply:
Even more info: https://stackoverflow.com/questions/3637660/how-to-exclude-file-only-from-root-folder-in-git

Reply:
Thanks for the patience 🙇
I learned something new today 🎉